### DISCUSSION TITLE
Log custom field

### DISCUSSION THREAD

**Discussion by airliajsmith** [spatie/laravel-activitylog#811] [Q&A] [answered]
I have added a custom column in my activity table called 'type'. I've been trying to use tap and tapActivity, but no luck,

How do I get the value of an input and pass it into the tapActivity method? 

I tried this with no luck:

    public function tapActivity(Activity $activity, string $request)
    {
        $activity->description = "{$request}";
    }

**Comment by Gummibeer** [comment, marked as answer]
You have to get the value from `$this` (Model), `$activity` or via Laravel Service Container `app()->make(...)`.
As the `tapActivity()` method is called automatically you can't inject your own values.

**Comment by airliajsmith** [comment]
Awesome, thanks!
